env-defaults

```diff
@@ -368,7 +368,8 @@ MAX_POST_FILE_PARTS=100
 
 # The location on the file system to save the work items that will be mounted into the
 # containers for any services to read and save STAC catalogs.
-HOST_VOLUME_PATH=
+# Defaulting to /tmp to allow tests to pass in CI
+HOST_VOLUME_PATH=/tmp
```

Comment:
Using /tmp doesn't work with Mac OS and Docker, and for a new developer setting up harmony it would cause problems that might be difficult to debug. I think that's enough reason to not set it in the defaults, and instead override it in the test env.ts file if it just needs to be set for the tests.

Reply:
Fixed